docs/cpp/config-clang-mac.md

```diff
@@ -188,6 +184,11 @@ with this:
 ```json
     "group": "build",
 ```
+5. After the build succeeds, your program's output will appear in the integrated **Debug Console**.
+    ![screenshot of debug console output](images/clang-mac/helloworld-debug-console-output.png)
+
+If you would like it to show up in the terminal, you have to tell the task to also **run** the generated program, by adding `"&&",` and `"${fileDirname}/${fileBasenameNoExtension}"` to the end of the list of arguments in `tasks.json`.
```

Comment:
```suggestion
To show it in the terminal, add`"&&",` and `"${fileDirname}/${fileBasenameNoExtension}"` to the end of the arguments list in the `tasks.json` file. This will **run** the generated program in the task.
```